Makefile

```diff
@@ -551,7 +558,7 @@ all: $(ROM)
 
 clean:
 	$(RM) -r $(BUILD_DIR_BASE)
-	make -C src/s2d_engine clean
+	$(RM) -r src/s2d_engine/build
```

Comment:
doesnt this throw an error if that directory doesnt exist? either way, can you wrap a `ifeq ($(TEXT_ENGINE), s2dex_text_engine)` around this?

Reply:
`-r -f` is reproducible so it shouldn't error (If you look at the definition of `RM`, it should be `rm -f` already

Reply:
ah okay cool, still would be nice to have this be conditional, but not a blocker